tests/manage/z_cluster/cluster_expansion/test_add_capacity.py

```diff
@@ -93,6 +100,14 @@ def add_capacity_test(ui_flag=False):
     if config.ENV_DATA.get("encryption_at_rest"):
         osd_encryption_verification()
 
+    # verify device classes
+    ocs_version = version.get_semantic_ocs_version_from_config()
+    if ocs_version >= version.VERSION_4_14 and not is_cluster_upgraded():
```

Comment:
IIUC this check is applicable only for clusters freshly deployed as 4.14 and newer (so not for clusters upgraded from 4.13 to 4.14), right?
What if in the future the cluster will be updated from 4.14 to 4.15, would this check apply there? 
(But then it will complicate little bit more... what if it will be updated twice - 4.13 -> 1.14 -> 4.15...)